app/controllers/admin/articles_controller.rb

```diff
@@ -40,7 +49,7 @@ def set_article
   end
 
   def article_param
-    params.permit('title', 'publish_status','elements_attributes':['tag_name', 'element_data', 'sory_key'])
+    params.require(:article).permit('title', 'publish_status','elements_attributes':['tag_name', 'element_data', 'sory_key'])
```

Comment:
__[rubocop]__
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Space missing after colon. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)